app/javascript/packages/document-capture/components/acuant-capture.jsx

```diff
@@ -142,7 +142,7 @@ function AcuantCapture({
    */
   function forceUpload() {
     isForceUploading.current = true;
-    inputRef.current.click();
+    inputRef.current?.click();
```

Comment:
awww makes me miss coffeescript

Reply:
JavaScript has taken quite a bit from CoffeeScript in the past few years 😄